hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3APrefetchingInputStream.java

```diff
@@ -240,4 +242,56 @@ public void testRandomReadSmallFile() throws Throwable {
     }
   }
 
+  @Test
+  public void testStatusProbesAfterClosingStream() throws Throwable {
+    describe("When the underlying input stream is closed, the prefetch input stream"
+        + " should still support some status probes");
+
+    byte[] data = ContractTestUtils.dataset(SMALL_FILE_SIZE, 'a', 26);
+    Path smallFile = path("testStatusProbesAfterClosingStream");
```

Comment:
you can just use `path()` and one is built up from the method name. this is better as it avoids cut and paste bugs *and* if a build is parameterized, the parameterized string is used in the path (though tests fail if that string isn't a valid path any more)

Reply:
Ah my bad, should have used `methodPath()`. You meant `methodPath()` only, correct?